metrics/prometheus_exporter_test.go

```diff
@@ -40,8 +41,9 @@ func TestNewPrometheusExporter(t *testing.T) {
 			component:          testComponent,
 			backendDestination: prometheus,
 			prometheusPort:     9091,
+			prometheusHost:     "127.0.0.1",
```

Comment:
Maybe add a test for configuration from env?

Reply:
ok I can do that.